src/masternodes/rpc_vault.cpp

```diff
@@ -1217,7 +1217,9 @@ UniValue listauctionhistory(const JSONRPCRequest& request) {
         if (isMine && !(IsMineCached(*pwallet, key.owner) & ISMINE_SPENDABLE)) {
             return true;
         }
-
+        if (key.index < start.index){
+            return true;
+        }
```

Comment:
is this necessary? this will create a filter effect. https://github.com/JellyfishSDK/jellyfish/pull/1526#issuecomment-1147116680
listauctionhistory should provide filtration only for `owner|vaultId` from a given pagination point.
so `blockHeight, index` should only be for pagination.